cuML/CMakeLists.txt

```diff
@@ -161,20 +169,25 @@ add_dependencies(gpufaisslib faiss)
 set_property(TARGET faisslib PROPERTY IMPORTED_LOCATION ${FAISS_DIR}/libfaiss.a)
 set_property(TARGET gpufaisslib PROPERTY IMPORTED_LOCATION ${FAISS_DIR}/gpu/libgpufaiss.a)
 
+
 include_directories(src
-	    ${CMAKE_CURRENT_BINARY_DIR}/faiss/include
+  	    ${CMAKE_CURRENT_BINARY_DIR}/faiss/include
 	    ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}
         ${GTEST_DIR}/include
         ${MLPRIMS_DIR}/src
         ${MLPRIMS_DIR}/test
         ${CUTLASS_DIR}
         ${CUB_DIR}
+        ${ZLIB_INCLUDE_DIRS$},
+        # TODO : Fix the path
+        /home/mehdia/miniconda3/envs/cuml-0128/include
```

Comment:
Please update this path to use CONDA related env-vars to find magma headers. I understand that you've put a todo note above. Put explicitly stating it here so that we don't miss this before merge.